feat: add info about new health check (#1079)
shopware/docs: guides/hosting/installation-updates/cluster-setup.md

For the server setup, pay special attention to CPU speed. This applies to all servers (app, SQL, Elasticsearch, Redis). Usually, it is more optimal to choose a slightly stronger CPU. This has to be determined more precisely depending on the project and load. Experience has shown that systems with powerful CPUs finish processes faster and can release resources sooner.

## Health Check

{% hint style="info" %}

> This feature has been introduced with Shopware version 6.5.5.0

Reviewer: ```suggestion
This feature is available starting with Shopware version 6.5.5.0
```

{% endhint %}

> Use the Shopware-provided Health Check API (`/api/_info/health-check`) to monitor the health of your Shopware app server. It runs http status 200 when the Shopware Application is working and 50x when it is not.

Reviewer: ```suggestion
Use the Shopware-provided Health Check API (`/api/_info/health-check`) to monitor the health of your Shopware app server. It runs HTTP status `200` when the Shopware Application is working and `50x` when it is not.
```
